index.js

```diff
@@ -83,6 +99,133 @@ function attachToBody (options, req, reply, next) {
   })
 }
 
+const allowedTokens = ['[', ']', '{}']
```

Comment:
actually those are "notAllowedAsFirstTokens"

Reply:
Actually, since this variable was being used in two places, this name wouldn't make sense in both contexts. If I was going to keep it, I would rename it to `specialTokens`, but I decided to simply remove it, as I've replaced the calls to `.includes` with chained ORs.